examples/apps/presence-tracker/src/FocusTracker.ts

```diff
@@ -3,22 +3,34 @@
  * Licensed under the MIT License.
  */
 
-import { Signaler } from "@fluid-experimental/data-objects";
-import { IEvent } from "@fluidframework/common-definitions";
-import { TypedEventEmitter } from "@fluidframework/common-utils";
-import { ITinyliciousAudience } from "@fluidframework/tinylicious-client";
-import { IFluidContainer, IMember } from "fluid-framework";
-
-export interface IFocusTrackerEvents extends IEvent {
-	(event: "focusChanged", listener: () => void): void;
+import { EventEmitter } from "events";
+import { ISignaler, SignalListener } from "@fluid-experimental/data-objects";
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
+import { assert } from "@fluidframework/common-utils";
+import { InternalSignaler } from "./InternalSignaler";
+
+export interface IFocusTracker extends EventEmitter, ISignaler {
+	getFocusPresences(): Map<string, boolean>;
+	getFocusPresenceForUser(userId: string, clientId: string): boolean | undefined;
+	on(event: "focusChanged", listener: () => void);
+	getName(): unknown | undefined;
 }
 
 export interface IFocusSignalPayload {
 	userId: string;
 	focus: boolean;
 }
 
-export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
+/**
+ * ITinyliciousUser describes the default format for users created through the Tinylicious server.
+ * However, the specific details for a user may differ depending on the server.
+ */
+export interface ITinyliciousUser {
+	id: string;
+	name: string;
+}
+
+export class FocusTracker extends DataObject implements IFocusTracker {
```

Comment:
Choosing to make FocusTracker a DataObject means it becomes part of the persisted state of the container - I'm not sure if this is something we want?  At minimum it is a pretty significant change in the purpose of the demo.